pages/advanced/content/refs.md

```diff
@@ -14,7 +14,8 @@ reference.
 var x = 'normal code block...'
 ```
 
-```interactive {clickToRender: true, summary: 'An input in a form'}
+```
+interactive {clickToRender: true, summary: 'An input in a form'}
```

Comment:
Unfortunately not. The way this works is I need this to be parsed as a code block with the `lang` set to `interactive {clickToRender: true, summary: 'An input in a form'}`

We [use that](https://github.com/kentcdodds/glamorous-website/blob/master/components/interactive-markdown.js) to make an [interactive experience](https://rc.glamorous.rocks/advanced) for the user. It's really nice to be able to interleave these kinds of things inside the markdown. We also have a `callout` pragma, and I expect there to be more in the future as well.